scst-scan/tanzu-supply-chain/create-supply-chain-with-app-scanning.hbs.md

```diff
@@ -7,8 +7,6 @@ solution for Supply Chains Choreographer for Tanzu.
 
 This section describes what dependencies are needed to create and run a Tanzu Supply Chain Workload.
 
```

Comment:
Would it help to break out the list of packages into:
packages
Tanzu Supply Chain components